modules/node/migrations/1597733020691-add-watcher.ts

```diff
@@ -0,0 +1,31 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class addWatcher1597733020691 implements MigrationInterface {
+    name = 'addWatcher1597733020691'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`DELETE FROM "typeorm_metadata" WHERE "type" = 'VIEW' AND "schema" = $1 AND "name" = $2`, ["public","anonymized_onchain_transaction"]);
+        await queryRunner.query(`DROP VIEW "anonymized_onchain_transaction"`);
```

Comment:
Is this doing anything?

Reply:
At some point a view was removed from one of the entities & this is bringing the db schema up to date w that change